Lib/importlib/_bootstrap_external.py

```diff
@@ -359,6 +359,7 @@ def _write_atomic(path, data, mode=0o666):
 #     Python 3.11a1 3454 (compute cell offsets relative to locals bpo-43693)
 #     Python 3.11a1 3455 (add MAKE_CELL bpo-43693)
 #     Python 3.11a1 3456 (interleave cell args bpo-43693)
+#     Python 3.11a1 3457 (Change localsplus to a bytes object bpo-43693)
```

Comment:
I wonder if we're not using up too many magic numbers during pre-alpha...

Reply:
what are our options?

Reply:
> what are our options?

Going back to the first magic number for 3.11a1. Everything is for the same bpo issue it seems.